homeassistant/auth/providers/insecure_example.py

```diff
@@ -16,22 +16,18 @@
 })
 
 
-CONFIG_SCHEMA = auth.AUTH_PROVIDER_SCHEMA.extend({
+CONFIG_SCHEMA = AUTH_PROVIDER_SCHEMA.extend({
     vol.Required('users'): [USER_SCHEMA]
 }, extra=vol.PREVENT_EXTRA)
 
 
-class InvalidAuthError(HomeAssistantError):
```

Comment:
Let's restore this. There is no reason to share an exception class just because it has a similar name. This is an internal exception and should stay so.